state.go

```diff
@@ -980,6 +995,7 @@ func (m *Memberlist) aliveNode(a *alive, notify chan struct{}, bootstrap bool) {
 			state.DMax = a.Vsn[4]
 			state.DCur = a.Vsn[5]
 		}
+		m.encodeBroadcastNotify(a.Node, aliveMsg, a, notify)
```

Comment:
Why did this need to get moved. Does something in `encodeBroadcastNotify` use the state's version numbers?